jovian/utils/rcfile.py

```diff
@@ -72,3 +73,18 @@ def make_rcdata(filename, slug):
         }
     }
     return json.dumps(data)
+
+
+def get_project():
+    """Get cached project parameter"""
+    global _current_project
+    return _current_project
+
+
+def set_project(project, new_project):
```

Comment:
You can simplify it as:
```
if new_project or project:
    _current_project = project
```

But it may be better to call `set_project` after `get_project` is called so that the `new_project` arg can be removed from this function. You will probably have to move `set_project` back to the previous location.

Reply:
Yes, will make that change.